apple/build.sh

```diff
@@ -74,7 +74,8 @@ build() {
             -DCMAKE_SYSTEM_NAME=iOS \
             -DCMAKE_OSX_DEPLOYMENT_TARGET=$IOS_DEPLOYMENT_TARGET \
             -DCMAKE_APPLE_ARCH_SYSROOTS="$SYSROOT" \
-            -DCMAKE_OSX_ARCHITECTURES="x86_64"
+            -DCMAKE_OSX_ARCHITECTURES="x86_64" \
+            -DCMAKE_BUILD_TYPE="$BUILD_TYPE"
```

Comment:
I don’t know why, but cmake does not set CMAKE_BUILD_TYPE even if we pass build type via —config option